jest.config.js

```diff
@@ -30,6 +30,9 @@ module.exports = {
   transformIgnorePatterns: [`/node_modules/(?!${esModules})`],
   coverageReporters: ['text-summary', 'lcov'],
   testPathIgnorePatterns: ['e2e'],
+  testEnvironmentOptions: {
+    customExportConditions: [''],
```

Comment:
What is this for?

Reply:
it had to do with the `setupServer` from msw but, I never got that working anyway. I'll fiddle around with those tests some more and if this is needed, I'll add a comment

Reply:
Do we still need it?

Reply:
yes we do still need it for `setupServer` in our tests. More info here in migration guide
https://mswjs.io/docs/migrations/1.x-to-2.x/#cannot-find-module-mswnode-jsdom